src/js/angular/repositories/ontop-repo.directive.js

```diff
@@ -0,0 +1,300 @@
+import {getFileName} from "./controllers";
```

Comment:
You can move this function here. It's not used anywhere else.

Reply:
It is used in the controllers.

Reply:
In let name = **getFileName**(data.params.ruleset.value); in EditRepositoryCtrl